blockchain/src/main/java/com/tangem/blockchain/blockchains/cosmos/CosmosWalletManager.kt

```diff
@@ -97,7 +98,15 @@ class CosmosWalletManager(
                     tax(amount)?.let { feeValue += it }
                     Amount(value = feeValue, blockchain = wallet.blockchain)
                 }
-            )
+
+
+                return if (amounts.size == 3) {
```

Comment:
Кажется тут стоит обработать сценарий с другим количеством `amounts` 
```suggestion
                return when (amounts.size) {
                    1 -> Result.Success(TransactionFee.Single(amounts[0]))
                    3 -> Result.Success(TransactionFee.Choosable(amounts[0], amounts[1], amounts[2]))
                    else -> Result.Error(IllegalStateException("Illegal amounts size"))
                }
```

Reply:
готово в https://github.com/tangem/blockchain-sdk-kotlin/pull/218